contracts/Protocol.sol

```diff
@@ -20,15 +19,22 @@ contract Protocol is IProtocol, Ownable {
     }
 
     function setProtocolVersion(string calldata deploymentSubset, uint256 protocolVersion, uint256 asOfBlock) external onlyOwner {
-        if (deploymentSubsets[deploymentSubset].version == 0) {
+        if (!deploymentSubsets[deploymentSubset].exists) {
             require(asOfBlock == 0, "initial protocol version must be from block 0");
```

Comment:
this is a very strange requirement. no protocol existed as of block 0 so the requirement that we start with as of block 0 is superficial and goes against the common case usage of this API. On the contrary, it makes sense to keep in the history the true launch date of any deployment subset with the first asOfBlock meaning something...

to me it makes more sense to see all deployment subsets as automatically defined by default on protocol version 0. 

@vistra this is not for this PR maybe but if you agree can we open an issue to fix this?

Reply:
Yes, that makes sense.

Reply:
See PR #25